tests/priceEngine.test.ts

```diff
@@ -0,0 +1,148 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest'
+import { PriceEngine } from '../src/core'
```

Comment:
In tsconfig we added baseUrl as root. So you can reference the imports like this

```suggestion
import { PriceEngine } from 'src/core'
```